packages/core/database/lib/entity-manager.js

```diff
@@ -402,6 +402,21 @@ const createEntityManager = (db) => {
             continue;
           }
 
+          // delete previous relations
+          const where = { $or: [] };
+
+          for (const { field, related_type: relatedType, related_id: relatedId } of rows) {
+            const isMorphOne =
+              db.metadata.get(relatedType).attributes[field].relation === 'morphOne';
+            if (isMorphOne) {
+              where.$or.push({ related_type: relatedType, field, related_id: relatedId });
```

Comment:
I'm wondering if it would be better to group by type to allow doing an $in for each type instead of one condition per row 🤔

Reply:
Grouping by type **and** field. I thought about it and why not honestly. I hesitated because it adds some code complexity for maybe not so much gain. I see the gain in case there are many many ids. So maybe it's worth it after all. wdyt?

Reply:
I have the feeling what we would win in query we might loose in js. The only win I see is it condenses the query so will avoid the "too many arguments" error in most cases

Reply:
I updated the PR